### ISSUE TITLE
Web POM missing jsp-api ref causing eclipse errors

### ISSUE THREAD

**Issue by GoogleCodeExporter** [psi-probe/psi-probe#430]
```
What steps will reproduce the problem?
1.open project in eclips
2.
3.

What is the expected result?
jsp's should not have compiler errors

What happens instead?
eclipse has compiler errors


What version of Probe are you using?
2.4

What environment (browser version, Tomcat version, JVM version, server OS)?
windows jdk8 

eclipse Version: Kepler Service Release 2
Build id: 20140224-0627

Please provide any additional information below.
(Attach logs or stack traces as files instead of pasting the contents
here.)

added jsp-api to web/pom.xml

		<dependency>
			<groupId>javax.servlet.jsp</groupId>
			<artifactId>jsp-api</artifactId>
			<version>2.2</version>
			<scope>provided</scope>
		</dependency>

```

Original issue reported on code.google.com by `t...@outboundengine.com` on 21 Nov 2014 at 2:50

**Comment by hazendaz** [comment]
This issue was solved on PR #483.  @MALfunction84, you can close this issue.

**Comment by MALfunction84** [comment]
Duplicate of #482.
